lower.ts

```diff
@@ -1,16 +1,15 @@
-import * as AST from './ast';
```

Comment:
URGENT – this entire file got reformatted so the diff is very large and we can't efficiently review it. Please restore its earlier formatting.